sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -55,15 +55,14 @@ internal TableServiceClient(Uri endpoint, TableSharedKeyPipelinePolicy policy, T
             var endpointString = endpoint.ToString();
             var secondaryEndpoint = endpointString.Insert(endpointString.IndexOf('.'), "-secondary");
             HttpPipeline pipeline;
-            var acceptPolicy = new TableAcceptHeaderPipelinePolicy();
 
             if (policy == default)
             {
-                pipeline = HttpPipelineBuilder.Build(options, acceptPolicy);
+                pipeline = HttpPipelineBuilder.Build(options);
             }
             else
             {
-                pipeline = HttpPipelineBuilder.Build(options, policy, acceptPolicy);
+                pipeline = HttpPipelineBuilder.Build(options, policy);
             }
```

Comment:
I think you can skip the condition because we filter out nulls.  https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs#L84